share/picongpu/tests/CurrentDeposition/lib/python/test/CurrentDeposition/MainTest.py

```diff
@@ -0,0 +1,191 @@
+import openpmd_api as io
```

Comment:
This file misses a general deocumentation at the beginning explaining its purpose. (just few lines)

Reply:
done